snippets/product-variant-picker.liquid

```diff
@@ -0,0 +1,99 @@
+{% comment %}
+    Renders product variant-picker
+
+    Accepts:
+    - product: {Object} product object.
+    - block: {Object} passing the block information.
+    - update_url: boolean (optinal).
```

Comment:
Let's follow the existing formatting and add some more details so it's easy to understand why it'd be used
```suggestion
    - update_url: {Boolean} for the share button. If false the share url isn't updated when changing variant (optional).
```